jdbc/src/main/resources/interpreter-setting.json

```diff
@@ -122,6 +122,20 @@
         "defaultValue": "-1",
         "description": "Maximum of connection lifetime in milliseconds. A value of zero or less means the connection has an infinite lifetime.",
         "type": "number"
+      },
+      "zeppelin.jdbc.rowsFetchSize": {
+        "envName": null,
+        "propertyName": "zeppelin.jdbc.rowsFetchSize",
+        "defaultValue": "1000",
+        "description": "Number of rows fetched from the database.",
```

Comment:
instead of "Number of rows fetched from the database.", should it be "Number of rows fetched from executing the query."

Reply:
Thanks! Fixed.